gitpod/gitpod.Dockerfile

```diff
@@ -1,48 +1,35 @@
 # Doing a local shallow clone - keeps the container secure
 # and much slimmer than using COPY directly or making a
 # remote clone
-ARG BASE_CONTAINER="pandas/pandas-dev:latest"
 FROM gitpod/workspace-base:latest as clone
 
-# the clone should be deep enough for versioneer to work
-RUN git clone https://github.com/pandas-dev/pandas --depth 12 /tmp/pandas
-
-# -----------------------------------------------------------------------------
-# Using the pandas-dev Docker image as a base
-# This way, we ensure we have all the needed compilers and dependencies
-# while reducing the build time
-FROM ${BASE_CONTAINER} as build
-
-# -----------------------------------------------------------------------------
 USER root
-
-# -----------------------------------------------------------------------------
-# ---- ENV variables ----
-# ---- Directories needed ----
-ENV WORKSPACE=/workspace/pandas/ \
-    CONDA_ENV=pandas-dev
-
-# Allows this micromamba.Dockerfile to activate conda environments
-SHELL ["/bin/bash", "--login", "-o", "pipefail", "-c"]
-
-# Copy over the shallow clone
-COPY --from=clone --chown=gitpod /tmp/pandas ${WORKSPACE}
-
-# Everything happens in the /workspace/pandas directory
-WORKDIR ${WORKSPACE}
-
-# Build pandas to populate the cache used by ccache
-RUN git config --global --add safe.directory /workspace/pandas
-RUN conda activate ${CONDA_ENV} && \
-    python -m pip install -e . --no-build-isolation && \
-    python setup.py build_ext --inplace && \
-    ccache -s
-
-# Gitpod will load the repository into /workspace/pandas. We remove the
-# directory from the image to prevent conflicts
-RUN rm -rf ${WORKSPACE}
-
-# -----------------------------------------------------------------------------
-# Always return to non privileged user
-RUN chown -R gitpod:gitpod /home/gitpod/.cache/
+# Install base utilities
+RUN apt-get update \
+    && apt-get install -y build-essential \
+    && apt-get install -y wget \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install miniconda
+ENV CONDA_DIR /opt/conda
```

Comment:
Not opposed but whats the advantage of running conda within Docker versus just using docker for the virtualization?

Reply:
I guess to install new packages?

For some reason I can't do stuff like ``apt install`` in gitpod.
(sudo doesnt work for me)

In general, I think it's preferred to have stuff installed from conda over pip, too.

Reply:
Does apt-get install work?

Reply:
Also on the note of conda versus pip - conda is a pretty big install. Most images try to be small and easy to deploy; I would imagine this makes the image significantly larger than the base

Reply:
Are users expected to modify this environment? If so I think installing conda would be nice for ease of use. But since this is for development anyways and not a service having conda here would be OK to me

Reply:
> Does apt-get install work?

I don't think so?
It's been a while since I last used gitpod. Sorry for the late reply.

Reply:
> Are users expected to modify this environment? If so I think installing conda would be nice for ease of use. But since this is for development anyways and not a service having conda here would be OK to me

I tend to want to install new packages at least
(e.g. things like a profiler, or trying out different versions of dependencies, like numpydev, etc.)